packages/opds-web-client/src/__tests__/DataFetcher-test.ts

```diff
@@ -11,7 +12,17 @@ describe("DataFetcher", () => {
     mockFetch = stub().returns(new Promise<any>((resolve, reject) => {
       resolve({ status: 200 });
     }));
-    fetch = mockFetch;
+    // mockFetch = new Promise<any>((resolve, reject) => {
+    //   resolve({ status: 200 });
+    // });
+    fetchMock
+      .mock("test-url", mockFetch)
```

Comment:
Can we get rid of the stub with this library? It looks like you can do `fetchMock.mock("test-url", 200)`

Reply:
Yes, I was still trying to figure out the full API of this mock package.